sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipeline.java

```diff
@@ -31,15 +31,30 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.Iterators;
 
+import com.fasterxml.jackson.core.JsonGenerator;
```

Comment:
Please add a test in TestPipelineTest that goes through the args conversion with a matcher and asserts that they are the same instance. I know that TestDataflowPipelineRunnerTest utilizes this but we want to keep this explicit.